gpjax/gps.py

```diff
@@ -789,7 +778,7 @@ def mll(params: Dict):
             wx = params["latent"]
 
             # f(x) = μx  +  Lx wx
-            fx = μx + jnp.matmul(Lx, wx)
+            fx = μx + Lx @ wx
```

Comment:
I understand why we can't right now but it'd be cleaner if we didn't have to use `@` for matmuls.

Reply:
An issue is open for this on `JaxLinOp` (JaxGaussianProcesses/JaxLinOp/#2).